### PR TITLE
[Merged by Bors] - [CU-2t7r06b] Integration Tests: Dex Router

### PR DESCRIPTION
**Pull request by Dom-Roth**
## Issue
https://app.clickup.com/t/2t7r06b

## Description
Recration of PR: #1399 

## Checklist
- [x]   Implemented all tests
- [x]   Implemented all required test checks
- [x]   Cleaned up all code
- [x]   Added required documentation
- [x]   Ran the whole test suite without any failures
- [x]   Ran npm run lint without any warnings / errors
- [x]   Created / Updated ClickUp report

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2t7r06b Integration Tests: Dex Router](https://app.clickup.com/t/2t7r06b)

**Review by Juliansv — APPROVED** [review]

File: integration-tests/runtime-tests/test/tests/dexRouter/dexRouterTests.ts [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+    eth = 5;
+    usdt = 6;
+    usdc = 7;
+    dai = 9;
+    //sets the fee to 1.00%/Type Permill
+    fee = 10000;
+    baseWeight = 500000;
+  });
+
+  // before("Minting assets", async function () {
```
Comment (kollegian) [inline integration-tests/runtime-tests/test/tests/dexRouter/dexRouterTests.ts:45]: Should these before steps commented out?

**Review by kollegian — APPROVED** [review]

**Review by Juliansv — APPROVED** [review]

**Review by Juliansv — APPROVED** [review]

**Review by kollegian — APPROVED** [review]

**Pull request closed without merging**